bin/admin.js

```diff
@@ -0,0 +1,105 @@
+#! /usr/bin/env node
+
+process.config = require('../config');
+
+const Promise = require('bluebird');
+const co = Promise.coroutine;
+const _ = require('lodash');
+const ArgumentParser = require('argparse').ArgumentParser;
+const pjson = require('../package.json');
+const fs = require('fs');
+const csvParser = require('csv-parse');
+
+const db = require('../app/db.js');
+const MasterKey = require('../app/models/MasterKey.js');
+
+const parser = new ArgumentParser({
+  version: pjson.version,
+  addHelp: true,
+  description: 'Key Recovery Service admin tool'
+});
+
+const subparsers = parser.addSubparsers({
+  title: 'commands',
+  dest: 'cmd'
+});
+
+const importKeys = subparsers.addParser('import', { addHelp: true });
+importKeys.addArgument(
+  [ 'file' ],
+  {
+    action: 'store',
+    help: 'path to a CSV list of public keys'
+  }
+);
+
+const saveKeys = co(function *(xpubs) {
+  const keys = [];
+
+  for (const xpub of xpubs) {
+    if (!xpub.startsWith('xpub') || xpub.length !== 111) {
+      console.log(`${xpub} is not a valid public key. Skipping`);
```

Comment:
Is this how we want to log errors? If you look in the old KRS we specified a logfile for the whole app to log to: https://github.com/BitGo/key-recovery-service/blob/master/app/app.js

Reply:
Morgan is used for HTTP request logging, so it was (and will be) used for the main KRS server (which is an HTTP server for the platform to connect to). This script is intended to be used locally, so it's more akin to the signoffline.js script from v1 (https://github.com/BitGo/key-recovery-service/blob/master/bin/signoffline.js). Lots of console.log()'s there.